src/main/groovy/org/jetbrains/intellij/IntelliJPluginExtension.groovy

```diff
@@ -24,36 +26,13 @@ class IntelliJPluginExtension {
         return runClasspath
     }
 
-    @SuppressWarnings("GroovyUnusedDeclaration")
-    def publish(Closure c) {
-        publish.with(c)
-    }
-
     public static class Publish {
```

Comment:
I don't know how relevant it is, but my preference would be to have this be its own top-level class unless it somehow benefits from the nesting.